libs/design-system/web-component-card/src/lib/web-component-card.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="image">
+  <img alt="image" [src]="imageUrl()" />
```

Comment:
Please set `alt=""`. The image is purely presentational and the text `"image"` doesn't convey any information to the user about what it actually is showing. https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#decorative_images